src/platform/linux/mod.rs

```diff
@@ -15,6 +16,10 @@ mod dlopen;
 pub mod wayland;
 pub mod x11;
 
+/// Environment variable that indicates the X11 backend should be used
+/// even if Wayland is available. In this case XWayland will be used.
```

Comment:
Now, this could use a clearer documentation maybe something along the lines of

> Environment variable specifying which backend should be used on unix platform.
>
> Legal values are `x11` and `wayland`. If this variable is set only the named backend
> will be tried by winit. If it is not set, winit will try to connect to a wayland connection,
> and if it fails will fallback on x11.
>
> If this variable is set with any other value, winit will panic.

And refer to this in the main documentation for visibility. Maybe the crate root, of `EventsLoop` documentation? Not sure which place would be best...